app/models/invenio_rdm_record_converter.rb

```diff
@@ -6,6 +6,7 @@
 class InvenioRdmRecordConverter < Sufia::Export::Converter
   include Galtersufia::GenericFile::InvenioResourceTypeMappings
   include Galtersufia::GenericFile::KnownOrganizations
+  SUBJECT_SCHEMES = [:tag, :mesh, :lcsh]
```

Comment:
This might be going over the top, but I figure I would normally use a constant in this sort of situation so why not use it here.